scripts/kind-e2e/nginx-federated.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+set -e
+
+TEST_NS="test-ns"
+export KUBECONFIG=$(echo $(git rev-parse --show-toplevel)/output/kind-config/local-dev/kind-config-cluster{1..3} | sed 's/ /:/g')
+kubectl config use-context cluster1
+
+echo "Creating a test namespace ${TEST_NS}."
+kubectl create ns ${TEST_NS}
+
+echo "Creating a ngnix application resources in namespace ${TEST_NS}."
+cat <<EOF | kubectl apply -n ${TEST_NS} -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: federated-web-file
+data:
+  index.html: "Hello from Kubernetes Cluster Federation!"
+EOF
+
+cat <<EOF | kubectl apply -n ${TEST_NS} -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: federated-nginx
+  labels:
+    app: federated-nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: federated-nginx
+  template:
+    metadata:
+      labels:
+        app: federated-nginx
+    spec:
+      containers:
+      - image: nginx:alpine
+        name: federated-nginx
+        volumeMounts:
+        - name: federated-web-file
+          mountPath: /usr/share/nginx/html/
+      volumes:
+        - name: federated-web-file
+          configMap:
+            name: federated-web-file
+EOF
+
+cat <<EOF | kubectl apply -n ${TEST_NS} -f -
```

Comment:
I believe you can also do:

kubectl apply -n ${TEST_NS} -f - <<EOF

(I haven't tried though..)

Reply:
Done